src/main/drivers/pwm_output_dshot_shared.c

```diff
@@ -230,6 +230,10 @@ FAST_CODE void pwmDshotSetDirectionOutput(
 #endif
 );
 
+#ifdef USE_DSHOT_TELEMETRY
+FAST_CODE void pwmDshotSetDirectionInput(motorDmaOutput_t * const motor);
```

Comment:
- in general, refactoring the order of function definitions is preferred to the use of forward declarations;
- in this case, neither should be needed, as the declaration is already exported in `src/main/drivers/pwm_output_dshot_shared.h`.

Reply:
Agreed, removed them.  I wanted to keep the changes easy to follow on a per-commit basis.  Added another commit to do this.